SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -207,4 +207,20 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 
 	UPROPERTY(EditAnywhere, config, Category = "Load Balancing", meta = (EditCondition = "bEnableUnrealLoadBalancer"))
 	TSubclassOf<class UAbstractLBStrategy> LoadBalanceStrategy;
+
+	UPROPERTY(EditAnywhere, Config, Category = "Replication", meta = (DisplayName = "Use RPC Ring Buffers"))
+	bool bUseRPCRingBuffers;
+
+	UPROPERTY(EditAnywhere, Config, Category = "Replication", meta = (EditCondition = "bUseRPCRingBuffers", DisplayName = "Default RPC Ring Buffer Size"))
+	uint32 DefaultRPCRingBufferSize;
+
+	/** Overrides default ring buffer size. */
+	UPROPERTY(EditAnywhere, Config, Category = "Replication", meta = (EditCondition = "bUseRPCRingBuffers", DisplayName = "RPC Ring Buffer Size Map"))
+	TMap<ERPCType, uint32> RPCRingBufferSizeMap;
```

Comment:
Not sure how to limit the key to client/server reliable/unreliable and multicast (currently also allows setting the key to "Invalid" and "Cross Server", which looks odd but doesn't really affect anything).